scripts/load_efficientnet.py

```diff
@@ -32,6 +32,9 @@ def _get_model_by_name(name, *args, **kwargs):
         'efficientnet-b3': efficientnet.keras.EfficientNetB3,
         'efficientnet-b4': efficientnet.keras.EfficientNetB4,
         'efficientnet-b5': efficientnet.keras.EfficientNetB5,
+        'efficientnet-b6': efficientnet.keras.EfficientNetB6,
+        'efficientnet-b7': efficientnet.keras.EfficientNetB7,
```

Comment:
```suggestion
        'efficientnet-b7': efficientnet.keras.EfficientNetB7,
        'efficientnet-b8': efficientnet.keras.EfficientNetB8,
```
missing **EfficientNetB8**